rita/src/rita_common/dashboard/network_endpoints.rs

```diff
@@ -38,3 +43,58 @@ pub fn set_settings(
 
     JsonStatusResponse::new(Ok("New settings applied".to_string()))
 }
+
+pub fn wipe(_req: HttpRequest) -> Result<HttpResponse, Error> {
+    if cfg!(not(debug_assertions)) {
+        return Ok(HttpResponse::NotFound().finish());
+    }
+
+    // Clean up existing WG interfaces
+    match cleanup() {
+        Ok(_) => trace!("Cleanup success!"),
+        Err(e) => {
+            warn!("Unable to complete cleanup: {:?}", e);
```

Comment:
Unable to complete cleanup of *what*?

Reply:
Also, prepend each message with `wipe: ` so that people know where this happens, as the logs only show the origin of a message up to module level.

Reply:
I mean, we don't want anyone to confuse the warning with something truly alarming.

Reply:
This cleanup is called at the beggining of client/exit which wipes wg interfaces. Good point on logging.